HHS/OPRE-OPS
# OPS-1588 CAN Review section

## What changed

Implements the CAN Review accordion for sending drafts-to-planning for approval.
 * Review Can Accordion
 * Can Summary Card
 * After Approval Toggle
 * Portfolios Tag

NOTE: the "CANS Outside Division Tag" and filtering the CAN cards by division have been removed after UX review and design change.

## Issue

#1588 

## How to test

* For an agreement with Draft BLIs, go to the BLIs tab and then click "Plan or Execute Budget Lines" to get the page.
* Select "Change Draft Budget Lines to Planned Status" and select some BLIs
* Verify the CAN Review section below contains the CAN funding summary card, the After Approval toggle, and the Portfolios tags

## Screenshots

![image](https://github.com/HHS/OPRE-OPS/assets/127965976/6bb2b284-723c-4e56-a145-e38f3f338791)

## Definition of Done Checklist


## Links

_If relevant, e.g. for a link to a piece of markdown documentation_
